pkg/config/config.go

```diff
@@ -36,15 +36,18 @@ func GetUserData() map[string]string {
 //	  so, a very natural reaction to this is checking the next chain variables yourself or leaving it blank - which creates a ton of technical debt and code duplication, or unpredictable behaviour for the user
 func GetServerUrl() string {
 	viper.AutomaticEnv()
-	if flagVal := viper.GetString(constants.ArgServerUrl); flagVal != "" {
-		// 1. Above all, use if the global flag is set
-		return flagVal
+	if flagVal := viper.GetString(constants.ArgServerUrl); viper.IsSet(constants.ArgServerUrl) {
+		// 1. Above all, use global flag val
+		if flagVal != "dns.de-fra.ionos.com" {
```

Comment:
maybe you can use !strings.Contains and use dns.DefaultIonosServerUrl instead of defining here "dns.de-fra.ionos.com" .

Reply:
can't use that constant because it would lead to circular imports :( but yeah, I will move it to our constants pkg so I can use it here.